x-pack/plugins/cases/public/types.ts

```diff
@@ -102,6 +105,10 @@ export interface CasesUiStart {
       find: (query: CasesFindRequest, signal?: AbortSignal) => Promise<Cases>;
       getCasesStatus: (query: CasesStatusRequest, signal?: AbortSignal) => Promise<CasesStatus>;
       getCasesMetrics: (query: CasesMetricsRequest, signal?: AbortSignal) => Promise<CasesMetrics>;
+      bulkGet: <Field extends keyof CaseResponse = keyof CaseResponse>(
+        params: CasesBulkGetRequestCertainFields<Field>,
```

Comment:
I noticed the type in `x-pack/plugins/cases/server/client/cases/client.ts` is actually

```
bulkGet<Field extends keyof CaseResponse = keyof CaseResponse>(
  params: CasesBulkGetRequestCertainFields<Field | 'id' | 'version' | 'owner'>
): Promise<CasesBulkGetResponseCertainFields<Field | 'id' | 'version' | 'owner'>>;
```

Why only `<Field>` vs `<Field | 'id' | 'version' | 'owner'>`?